claim/README.md

```diff
@@ -116,6 +116,67 @@ For the connection process to work, the contents of `/var/lib/netdata` _must_ be
 restarts using a persistent volume.  See our [recommended `docker run` and Docker Compose
 examples](/packaging/docker/README.md#create-a-new-netdata-agent-container) for details.
 
+#### Known issue on systems with enabled seccomp
+
+The nodes running on the following hosts **cannot be claimed**:
+
+- `libseccomp` version less than v2.3.3.
+- Docker version less than v18.04.0-ce.
+- the kernel is configured with CONFIG_SECCOMP enabled.
+
+To check if your kernel supports `seccomp`:
+
+```cmd
+# grep CONFIG_SECCOMP= /boot/config-$(uname -r) 2>/dev/null
+CONFIG_SECCOMP=y
```

Comment:
This only works on some systems, not all. There’s not really any universal way to check kernel config options for the running kernel on a given system (though on most distros you can `modprobe configs && zgrep <config_option> /proc/config.gz`).

However, this is unlikely to be worth checking. Pretty much every Linux distro that is expected to possibly run a desktop environment has seccomp enabled in their kernels because it’s a required security feature for Chrome and Firefox on Linux.

Reply:
I copied that check from https://docs.docker.com/engine/security/seccomp/

Reply:
ok, updated to

> grep CONFIG_SECCOMP= /boot/config-$(uname -r) 2>/dev/null ||  zgrep CONFIG_SECCOMP  /proc/config.gz 2>/dev/null

works on both debian9 and manjaro

Reply:
Expected error log messages on such systems

> No write permission in claiming working directory /var/lib/netdata/cloud.d
>
> netdata.public.unique.id is not readable. Please make sure you have rights to read it (Filename: /var/lib/netdata/registry/netdata.public.unique.id).

I think I just add them and say if you see them then...

In that case yeah, we probably don't need that check.